annotate/module.ily

```diff
@@ -112,20 +112,37 @@
              #{ \once \override #item #'input-annotation = #props #}))))
          #{
           #tweak-command
-          #(if (assq-ref props 'footnote-offset)
-          ;; If offset present, add automatic footnote
+          #(cond
```

Comment:
Using `cond` means there can be *either* a footnote *or* a ballon text. Is this intended/a necessity?